### PR TITLE
[quant][graph] Graph mode quantization support for sigmoid

### PR DESCRIPTION
**Pull request by supriyar**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#36622 [quant][graph] Graph mode quantization support for sigmoid**
* #36552 [quant][graph] Add quantized batch_norm2d_relu to graph mode
* #36692 [quant][graph] Add quantized batch_norm2d support to graph mode
* #36691 [quant][graph] Add useQuantizable function

Summary:

Test Plan:
python test/quantization/test_quantize_script.py test_swap_dequantize_all_ops

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21075255](https://our.internmc.facebook.com/intern/diff/D21075255)

### CODE REVIEW

**Review by jerryzh168 — APPROVED** [review]

File: test/quantization/test_quantize_script.py [hunk 1]
```diff
@@ -1464,6 +1464,7 @@ def forward(self, x):
                 x = torch.max(x)
                 x = torch.min(x)
                 x = torch.mean(x)
+                x = torch.sigmoid(x)
```
Comment (jerryzh168) [inline test/quantization/test_quantize_script.py:1467]: looks like we have sigmoid module as well: https://pytorch.org/docs/stable/nn.html?highlight=sigmoid#torch.nn.Sigmoid
could you add a test for that?

**Review by jerryzh168 — APPROVED** [review]
